.github/support.md

```diff
@@ -0,0 +1,24 @@
+## Are you having trouble getting started?
+If you haven't already, please checkout our [documentation](https://docs.bugsnag.com/platforms/flutter/) for guides, references and tutorials.
+
+Or, if you wish you can [contact us directly](mailto:support@bugsnag.com) for assistance on integrating Bugsnag into your application, troubleshooting an issue or a question about our supported features.
+
+When contacting support, please include as much information as necessary, including:
+
+- example code snippet
+- steps to reproduce
+- expected/actual behaviour 
+
+* Bugsnag version:
+* Dart version:
```

Comment:
Same as above - the Dart version is defined by which version of the Flutter SDK is being used